content/libraries/ruby.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Ruby
+description: The Official Top.gg ruby library
+---
+
+# Ruby Library
+
+This is our official Ruby Library for top.gg, if you have any issues/bugs please submit an issue on our github.
+
+- [GitHub Link](https://github.com/top-gg/ruby-sdk)
+
+## Documentation
+
+- [Api Reference](rubydoc.info/gems/topgg/)
+
+## Installation
+
+```bash
+
+gem install topgg
+
+```
+
+## Autoposting Statistics
+
+```ruby
+require 'topgg'
+require 'discordrb'
+
+bot = Discordrb::Bot.new token: "TOKEN"
+
+client = Topgg.new("AUTH_TOKEN", "BOTID")
+ bot.ready do |event|
+ client.auto_post_stats(bot) # The discordrb bot client.
+end
+bot.run
+```
+
+## Example Usage
+
+Here's an example on how to use the library!
+
+```ruby
+require 'topgg'
+bot = Discordrb::Bot.new token: "TOKEN"
+
+client = Topgg.new("AUTH_TOKEN", "BOTID")
+# check if the speicified user id voted or not.
+p client.voted?("1234")
+```
+
+You can check out the [API Reference](rubydoc.info/gems/topgg/) to know about the other methods!
```

Comment:
URLs in markdown need to be fully qualified otherwise they're relative to the current page and don't work